### ISSUE TITLE
ipfs id --offline <peerid> does not work

### ISSUE THREAD

**Issue by lidel** [ipfs/go-ipfs#8481]
### Checklist

- [X] This is a bug report, not a question. Ask questions on [discuss.ipfs.io](https://discuss.ipfs.io).
- [X] I have searched on the [issue tracker](https://github.com/ipfs/go-ipfs/issues?q=is%3Aissue) for my bug.
- [X] I am running the latest [go-ipfs version](https://dist.ipfs.io/#go-ipfs) or have an issue updating.

### Installation method

built from source

### Version

```Text
0.10.0
```

### Config

```json
(default)
```

### Description

`ipfs id --peerid-base base36 --offline <peerid>` does not work – it requires connection and hangs until one is established.

## Rationale why this should be prioritized 

We would like to have an easy way for converting peerids to a CIDv1 in arbitrary multibase.
Right now, there is no universal way that works for all peerid types and is easy to use.
Example: https://discuss.ipfs.io/t/command-line-to-transform-base58-peerid-to-subdomain-friendly-base36/12330/3?u=lidel

`ipfs id --peerid-base base36 --offline $PEERID  |  jq --raw-output .ID` nearly does the job, but it does not respect `--offline` flag and requires connection to the peer.

**Comment by schomatis** [comment]
Agreed with triage and working on this. It seems to me that if we only need encoding conversion this should live in an independent command maybe. Regardless, I'll implement it here and we can decouple it later if we want.

**Comment by schomatis** [comment]
@lidel There is no `--offline` option in this command (not even in the form of documentation or a fixme, just an error message). My interpretation of it and please correct/complete if wrong is: do not attempt to connect to the peer (no matter if the daemon is running or not) and just print whatever information we have in the peer address book (which will likely be nothing except for the converted peer ID).

**Comment by schomatis** [comment]
(Is the daemon's `--offline` option supposed to apply here? I'm getting a collision when adding a local `--offline` option so I'm assuming there is a global one up in the hierarchy which is the one referred in this issue.)

**Comment by schomatis** [comment]
The global  offline option is in:

https://github.com/ipfs/go-ipfs/blob/91c52657166bcf86f2476926e4fe56694dc26562/core/commands/root.go#L25
